tests/smoke/base.sh

```diff
@@ -70,9 +70,12 @@ $COMMAND_PREFIX pre-commit install
 
 $COMMAND_PREFIX pre-commit run --all-files --verbose
 
-$COMMAND_PREFIX make docs-build
+$COMMAND_PREFIX make docs-build 
```

Comment:
for all the places that uses `make` in this file  ... you will need to change them .. and add an `if` that will check if it uses make or makim and call the correct command.

remember that the `make target's name` here use `-` but `makim` uses `.`.